Source/Core/DolphinTool/ConvertCommand.cpp

```diff
@@ -93,16 +93,26 @@ int ConvertCommand::Main(const std::vector<std::string>& args)
   // --scrub
   const bool scrub = static_cast<bool>(options.get("scrub"));
 
+  if (scrub && format != DiscIO::BlobType::RVZ && volume->IsDatelDisc())
```

Comment:
Doesn't make sense. Should be:

```suggestion
  if (scrub && (format == DiscIO::BlobType::RVZ || volume->IsDatelDisc()))
```

Or, you could make the RVZ part just a warning. Scrubbing RVZ does work correctly, it's just that it saves almost no space. (Keep the Datel part an error.)